packages/api-v2/src/AuthProvider.tsx

```diff
@@ -23,14 +23,12 @@ type AuthContextType = {
     subscribe: <T extends TSocketSubscribableEndpointNames>(
         name: T,
         payload?: TSocketRequestPayload<T>
-    ) =>
-        | {
-              subscribe: (
-                  onData: (response: any) => void,
-                  onError: (response: any) => void
-              ) => { unsubscribe?: VoidFunction };
-          }
-        | undefined;
+    ) => {
+        subscribe: (
+            onData: (response: any) => void,
```

Comment:
```suggestion
            onData: (response: unkown) => void,
```
@wojciech-deriv, doesn't `unknown` work here? so that the consumer is at least forced to check for the type before using it? 🤔